app/controllers/content_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ContentController < ApplicationController
+  def index
+    @content = get_content
+  end
+
+private
+
+  def get_content
+    response = metrics_service.content_summary(date_range: date_range,
```

Comment:
```MetricsServic``` is doing to much. It might be better to move
into ServiceObjects that do only one thing.

I would be keen to be able to rewrite the ```index``` action into:

```ruby
def action
  response = FindContent.call(params)
  @content = ContentItemsPresenter.new(response)
end
```

```FindContent``` service would live in ```/services/find_content.rb```, and would 
parse the parameters